v2/dtos/common/version.go

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package common
+
+// VersionRequest defines the request associated with the /version endpoint.
+// This object and its properties correspond to the VersionRequest object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/VersionRequest
+type VersionRequest struct {
+	RequestID string `json:"requestId" validate:"required,uuid"`
+}
+
+// VersionResponse defines the latest version supported by the service.
+// This object and its properties correspond to the VersionResponse object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/VersionResponse
+type VersionResponse struct {
```

Comment:
Can you please add the VersionSdkResponse that extends this response DTO with 
```
SdkVersion      string `json:"sdk_version"`
```

Both Device and App SDks will need this. Thx
@rsdmike FYI.